packages/enzyme/src/RSTTraversal.js

```diff
@@ -34,6 +34,7 @@ export function childrenOfNode(node) {
 export function hasClassName(node, className) {
   let classes = propsOfNode(node).className || '';
   classes = String(classes).replace(/\s/g, ' ');
+  if (className instanceof RegExp) return className.test(classes);
```

Comment:
`instanceof` isn't a reliable means of testing things for any builtin; please use the https://www.npmjs.com/package/is-regex package instead.

Reply:
Done